lopper/assists/xlnx_overlay_dt.py

```diff
@@ -138,6 +138,19 @@ def xlnx_generate_overlay_dt(tgt_node, sdt, options):
         sys.exit(1)        
 
     pl_node = None
+    has_valid_pl = None
+    has_pl = None
+
+    for node in root_sub_nodes:
+        if node.name == "amba_pl":
+            get_sub_node = node.subnodes()
+            for node in get_sub_node:
+                if node.propval('reg') != ['']:
+                    has_valid_pl = True
+                    break
+        if has_valid_pl == True:
```

Comment:
if has_valid_pl:
is sufficient, no need to check for == True

Reply:
Will fix in v2.